addons/binding/org.openhab.binding.onkyo/src/main/java/org/openhab/binding/onkyo/internal/OnkyoAlbumArt.java

```diff
@@ -15,10 +15,9 @@
 import java.net.URLConnection;
 import java.util.Arrays;
 
-import javax.xml.bind.DatatypeConverter;
-
 import org.apache.commons.io.IOUtils;
 import org.eclipse.jdt.annotation.NonNull;
+import org.eclipse.smarthome.core.util.HexUtils;
```

Comment:
Should you import this in the manifest?

Reply:
You already added it in https://github.com/openhab/openhab2-addons/pull/3431 ;-)